src/VssCommandProcessor.cpp

```diff
@@ -437,14 +435,7 @@ string VssCommandProcessor::processQuery(const string &req_json,
     string action = root["action"].as<string>();
     logger->Log(LogLevel::VERBOSE, "Receive action: " + action);
 
-    if (action == "get") {
-        response = processGet2(channel, root);
```

Comment:
Why disabling get requests?

Reply:
strange here... I do not know what git done!